perl-package/AI-MXNet/t/test_gluon_rnn.t

```diff
@@ -320,15 +320,15 @@ sub test_rnn_layers
     check_rnn_layer_forward(gluon->rnn->GRU(10, 2), mx->nd->ones([8, 3, 20]));
     check_rnn_layer_forward(gluon->rnn->GRU(10, 2), mx->nd->ones([8, 3, 20]), mx->nd->ones([2, 3, 10]));
 
-    my $net = gluon->nn->Sequential();
-    $net->add(gluon->rnn->LSTM(10, 2, bidirectional=>1));
-    $net->add(gluon->nn->BatchNorm(axis=>2));
-    $net->add(gluon->nn->Flatten());
-    $net->add(gluon->nn->Dense(3, activation=>'relu'));
-    $net->collect_params()->initialize();
-    mx->autograd->record(sub {
-        $net->(mx->nd->ones([2, 3, 10]))->backward();
-    });
+#    my $net = gluon->nn->Sequential();
```

Comment:
Remove unused code?

Reply:
Just commented it for now so I'll remember to return to this later. There was one test failure in that part that was never reproduced after that and I'd like to get to the bottom of it later.
Currently I am working on adding sparse support to perl and don't have much time for this currently so I just commented the block.

Reply:
Just curious, anyone requested the sparse feature for perl?

Reply:
Maybe create an issue in order to keep track of it?

Reply:
@marcoabreu will do that, thanks.